src/azul/__init__.py

```diff
@@ -1442,7 +1442,7 @@ def docker_registry(self) -> str:
         'docker.io/library/docker:24.0.2-dind',
         'docker.io/library/python:3.9.17-bullseye',
         'docker.io/lmenezes/cerebro:0.9.4',
-        'docker.io/ucscgi/azul-pycharm:2022.3.3-3',
+        f'docker.io/ucscgi/azul-pycharm:{os.environ.get("azul_docker_pycharm_version")}',
```

Comment:
```suggestion
        f'docker.io/ucscgi/azul-pycharm:{os.environ.get(self.docker_pycharm_version)}',
```

And then this attribute will need to be an `@property` or preferrably,  a `@cached_property`.